From 1c1293b3e31a9e4156ae7a0d099951c56f582da6 Mon Sep 17 00:00:00 2001
From: Claudio Matsuoka <cmatsuoka@gmail.com>
Date: Sun, 3 Jun 2012 13:36:38 -0300
Subject: [PATCH] Make test script a bit more portable

Replaced mktemp which can be different/absent in some systems with
static filenames which should be good enough for our tests.

Signed-off-by: Claudio Matsuoka <cmatsuoka@gmail.com>
---
 run-tests.sh |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/run-tests.sh b/run-tests.sh
index 26d2581..e6e1e3b 100755
--- a/run-tests.sh
+++ b/run-tests.sh
@@ -4,7 +4,7 @@ LC_ALL=POSIX
 export LC_ALL
 
 TESTDIR=tests
-OUTPUT=`mktemp`
+OUTPUT=.test-output.txt
 LOGFILE=tests.log
 CMD=./figlet
 FONTDIR="$1"
@@ -67,7 +67,7 @@ run_test 018 "tlf2 overlap centered mode" "$cmd -oc -f tests/emboss"
 run_test 019 "tlf2 full-width flush-left right-to-left mode" \
   "$cmd -WRl -f tests/emboss"
 run_test 020 "specify font directory" \
-  "X=`mktemp -d`;cp fonts/script.flf \$X/foo.flf;$cmd -d\$X -ffoo;rm -Rf \$X"
+  "X=.t;mkdir \$X;cp fonts/script.flf \$X/foo.flf;$cmd -d\$X -ffoo;rm -Rf \$X"
 run_test 021 "paragraph mode long line output" "$cmd -p -w250"
 run_test 022 "short line output" "$cmd -w5"
 run_test 023 "kerning paragraph centered mode (small)" "$cmd -kpc -fsmall"
-- 
1.7.10

